Replace helper_state with index into per-aggregator storage (#232)
ietf-wg-ppm/ppm-specification: draft-gpew-priv-ppm.md

Once a set of clients have uploaded their reports to the leader, the leader can
send them to the helpers to be verified and aggregated. In order to enable the
system to handle very large batches of reports, this process can be performed
incrementally. Verification of a set of reports is referred to as an aggregation

> job. Each aggregation job is associated with at most one PPM task, and a PPM

Reviewer: "at most one" suggests an aggregation job could be associated with zero PPM tasks.
```suggestion
job. Each aggregation job is associated with exactly one PPM task, and a PPM
```

~~~
Leader                                                 Helper

Aggregate request (Reports 1-10, Job = N) --------------->  \
<----------------------------- Aggregate response (Job = N) | Reports
Aggregate request (continued, Job = N) ------------------>  | 1-10
<----------------------------- Aggregate response (Job = N) /

> Aggregate request (Reports 11-20, Job = N) -------------->  \

Reviewer: I think the idea is that reports 11-20 would be job `M` where `M !=N`.

> Aggregate request (Reports 11-20, Job = N) -------------->  \
> <----------------------------- Aggregate response (Job = N) | Reports
> Aggregate request (continued, Job = N) ------------------>  | 11-20
> <----------------------------- Aggregate response (Job = N) /

Author: ```suggestion
Aggregate request (Reports 11-20, Job = M) -------------->  \
<----------------------------- Aggregate response (Job = M) | Reports
Aggregate request (continued, Job = M) ------------------>  | 11-20
<----------------------------- Aggregate response (Job = M) /
```
